zio-http/src/main/scala/zhttp/http/Middleware.scala

```diff
@@ -38,6 +38,14 @@ sealed trait Middleware[-R, +E, +AIn, -BIn, -AOut, +BOut] { self =>
    */
   final def apply[R1 <: R, E1 >: E](http: Http[R1, E1, AIn, BIn]): Http[R1, E1, AOut, BOut] = execute(http)
 
+  /**
+   * Makes the middleware resolve with a constant app
+   */
+  final def as[R0, E0, A, B](
+    http: Http[R0, E0, A, B],
+  ): Middleware[R0, E0, AIn, BIn, A, B] =
+    Middleware.Constant(http)
```

Comment:
This will ignore the current middleware. We need to implement this operator using `map`